### PR TITLE
Improve code

### PR DESCRIPTION
**Pull request by lntuition**
This is minor patch to improve code better. This patch is not related to any feature.

* [x] refactor execute in `queryset` to remove duplicate code
* [x] remove duplicate assignment at init relation
* [x] remove duplicate Exception in relational field

### CODE REVIEW

File: tortoise/queryset.py [hunk 1]
```diff
@@ -479,7 +475,8 @@ def get(self, *args: Q, **kwargs: Any) -> QuerySetSingle[MODEL]:
         """
         queryset = self.filter(*args, **kwargs)
         queryset._limit = 2
-        queryset._get = True
+        queryset._single = True
+        queryset._raise_does_not_exist = True
```
Comment (grigi) [inline tortoise/queryset.py]: Should you not also set it for `get_or_none`?

Comment (lntuition) [reply tortoise/queryset.py]: I think `get_or_none` do not raise `DoesNotExist`. it will return None

**Comment by lntuition** [comment]
Also I don't know why CI failed with pypy...

File: tortoise/queryset.py (hunk 1 above)
Comment (grigi) [reply tortoise/queryset.py]: Doh, yet, you are right. It should however raise an exception if more than one record gets returned. I got confused about that.

**Comment by grigi** [comment]
The one test is a bit fragile, and timing dependant. Sometimes Travis has a hiccup on that one test. I restarted it, it should just go away soon.

**Pull request merged**